content/serverkit/docs/modules.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Modules"
+date: 2017-09-19T10:52:18-07:00
+draft: false
+listdescription: "Explore the modules that make up the WorldWind Server Kit."
+---
+
+## Modules
+
+---
+
+[WorldWind GeoServer Application](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-geoserver)
+
+[WorldWind GeoServer Distribution](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-geoserver-dist)
+
+[WorldWind GeoPackage Extension](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-gs-geopkg)
+
+[WorldWind GeoPackage Module](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-gt-geopkg)
+
+[WorldWind GeoServer WMS Module](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-gs-wms)
```

Comment:
There is an additional module to add: WorldWind GeoServer Viewer (or Explorer, TBD). It's under development in the feature/#107-worldwind_viewer branch